src/Maths/Silk.NET.Maths/Scalar.MathFPort.MissingMethods.cs

```diff
@@ -0,0 +1,382 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Runtime.CompilerServices;
+using static Silk.NET.Maths.Helper;
+
+namespace Silk.NET.Maths
+{
+    partial class Scalar
+    {
+        ///////////////////////////////////////////////// BEGIN GO CODE ///////////////////////////////////////////////
+
+        // Adapted from https://golang.org/src/math/atanh.go
```

Comment:
This is all that's needed for the accompanying comments for the second license headers.

Not a comment.